resources/validations/README.md

```diff
@@ -17,23 +17,36 @@ project structure
 To validate xml files using schematron
 ---
 
-example
+*Prerequesite
+if you haven't done it previously: to add the needed dependencies (declared by .gitmodules), run the following:*
+
+`-f` *\<required>* is the input file to be tested. ex: `-f test/demo/FedRAMP-SSP-OSCAL-Template.xml`
 
-`./validate_with_schematron.sh test/demo/FedRAMP-SSP-OSCAL-Template.xml`
+`-s` *\<optional>* schematron directory used to validate the file. Each .sch found will be compliled and generate a separate report. defaults to src/*.sch relative to this script.  ex: `-o ~/mySchematronDirectory`
```

Comment:
I know you say "schematron directory" and "defaults to src/*.sch", but the default is really `src` and you append the file glob, so you want to explain it should be `-s src` or `-s path/to/schematron` a little more explicitly. My $0.02.

Reply:
good catch of my copy pasta from previous example, i will remedy that.